apps/io-services-cms-webapp/src/watchers/on-legacy-service-change.ts

```diff
@@ -167,11 +114,59 @@ const legacyToCms = (
     )
   );
 
+const serviceWasPublished = (
+  item: LegacyService,
+  legacyServiceModel: ServiceModel
+): TE.TaskEither<Error, O.Option<boolean>> =>
```

Comment:
question: why are you returning an Optional? Can't we return a boolean directly?

Reply:
Changed in d34deed, now we map the method result into an Option when needed